src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -194,7 +195,7 @@ public static Integer parseTheme(Optional<String> theme) throws IllegalValueExce
         final ArrayList<String> validThemes = new ArrayList<String>(Arrays.asList(validThemeStrings));
```

Comment:
I don't think this belongs in the parser, maybe put it in UiManager, and access it here. Based on list of strings, generate `MESSAGE_INVALID_THEME`